crates/librqbit/src/torrent_state/live/mod.rs

```diff
@@ -1717,6 +1806,7 @@ impl PeerHandler {
         B: AsRef<[u8]> + std::fmt::Debug,
     {
         // TODO: this is just first attempt at pex - will need more sophistication on adding peers - BEP 40,  check number of live, seen peers ...
+        debug!("received PEX message with {} added peers and {} dropped peers", msg.added_peers().count(), msg.dropped_peers().count());
```

Comment:
every message is already logged, you don't need this?

Reply:
It was just for debugging - to understand what other peers are sending.
Interestingly some peers were sending completely empty PEX message with no added nor dropped peers, which is against spec.
Removed.